ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml

```diff
@@ -22,6 +22,7 @@ postsubmits:
         - --report-password-file=/etc/report/password.txt
         - --report-username=ci
         - --target=[images]
+        - --upload-via-pod-utils
```

Comment:
@josephca I am not sure about this change. Can you please rebase the pr

Reply:
@amitkrout Actually, this change was generated by running 'make jobs' and this change is already available in 'master' branch, thus the merge would ignore this change. Nonetheless, in order to keep the commit history clean, I created a new PR by rebasing it to master again - https://github.com/openshift/release/pull/15267